* Changing the environment installation name

fonsecareyna82 opened the pull request:
* Adding more information about the installation process

fonsecareyna82 committed:
* Changing the environment installation name
* Adding more information about the installation process

pconesa commented on scipioninstaller/installer.py:
```diff
@@ -301,6 +301,8 @@ def main():
         # Check Scipion home folder and create it if apply.
         solveScipionHome(scipionHome, dry, noAsk)
         scipionEnv = args.n
+        if not conda:
```
I'd check if scipionEnv ==SCIPION_ENV too

dmaluenda replied:
Me too and I would append the suffix 'env' to the environ name, resulting in `.scipion3env` in order **to avoid** having something like:         (only if virtualenv is used and no environ-name is passed)
```
scipion3         # SCIPION_HOME usually is scipion3
  ├── .scipion3  # virtualenv environ
  ├── scipion3   # Scipion launcher
(...)
  └──  whatever
```
too much scipion3 everywhere XD

dmaluenda commented on scipioninstaller/installer.py:
```diff
@@ -310,19 +312,54 @@ def main():
+        content.append(" ")
+        content.append("For more information about the installation errors that can appear when installing or ")
+        content.append("using Scipion go to: https://scipion-em.github.io/docs/docs/user/troubleshooting.html ")
+        tableLength = len(content[-2]) if len(content[-2]) > len(content[1]) else len(content[1])
```
Hi. If I understand well the propose of `tableLenght`. I suggest to make it inside the `createMessageInstallation` using something like
```
tableLenght = max([len(c) for c in content])
tableLenght = tableLenght + tableLenght % 2
```

fonsecareyna82 committed:
* Modifying the createMessageInstallation method

fonsecareyna82 committed:
* Modifying the createMessageInstallation method
* Checking if scipionEnv == SCIPION_ENV

fonsecareyna82 committed:
* Changing the installer version

pconesa committed:
CONDA ACTIVATION CMD added to the launcher
Fix installation in tcsh/csh shells ?? (hopefully)
Final messages lack utf-chars to simplify table

pconesa committed:
fix indentation

fonsecareyna82 committed:
* Fixing an error creating CONDA_ACTIVATION_CMD variable
  - This variable is not necessary with virtualenv

pconesa committed:
Avoid guessing conda initialization is CONDA_ACTIVATION_CMD is passed.
Make multiple dirs for scipionHome.
